packages/@okta/courage-for-signin-widget/README.md

```diff
@@ -0,0 +1,11 @@
+# @okta/courage-for-signin-widget
+
+Generates a custom version of Courage for the Sign-In Widget. 
+It's intend for internal developers and please open a ticket if any questions.
+
+## Steps
+
+0. make sure VPN is connected
```

Comment:
nit: Capitalize the beginning of each ordered list item

Reply:
+1